contracts/SplitEquippable/AdvancedNestableExternalEquip.sol

```diff
@@ -2,15 +2,12 @@
 
 pragma solidity ^0.8.16;
 
-import "@rmrk-team/evm-contracts/contracts/RMRK/equippable/RMRKNestingExternalEquip.sol";
+import "@rmrk-team/evm-contracts/contracts/RMRK/equippable/RMRKNestableExternalEquip.sol";
 
-contract AdvancedNestingExternalEquip is RMRKNestingExternalEquip {
-    constructor(
-        string memory name,
-        string memory symbol
+contract AdvancedNestableExternalEquip is RMRKNestableExternalEquip {
+    constructor(string memory name, string memory symbol)
         // Custom optional: additional parameters
-    )
```

Comment:
Another constructor that I think should be reverted back to the previous structure 🙈